extensions/sql-database-projects/src/models/project.ts

```diff
@@ -1321,6 +1370,72 @@ export class Project implements ISqlProject {
 		}
 	}
 
+	/**
+	 * Adds a value to project property, separated by semicolon if
+	 * one or more values are already present in project property
+	 * @param propertyName Name of property
+	 * @param valueToAdd Value to be added to project property
+	 */
+	private async addValueToProjectProperty(propertyName: string, valueToAdd: string) {
```

Comment:
I'd rename this to `addValueToCollectionProjectProperty` or something, to indicated that in assume specific format of the property value and any property won't do. Same for remove.